sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfigurationFacadeEjb.java

```diff
@@ -383,13 +384,23 @@ public void deleteAllExpiredFeatureConfigurations(Date date) {
 	public boolean isFeatureDisabled(FeatureType featureType) {
 
 		CriteriaBuilder cb = em.getCriteriaBuilder();
-		CriteriaQuery<Long> cq = cb.createQuery(Long.class);
+		CriteriaQuery<FeatureConfiguration> cq = cb.createQuery(FeatureConfiguration.class);
 		Root<FeatureConfiguration> root = cq.from(FeatureConfiguration.class);
 
-		cq.where(cb.and(cb.equal(root.get(FeatureConfiguration.FEATURE_TYPE), featureType), cb.isFalse(root.get(FeatureConfiguration.ENABLED))));
-		cq.select(cb.count(root));
+		cq.where(cb.and(cb.equal(root.get(FeatureConfiguration.FEATURE_TYPE), featureType)));
+		List<FeatureConfiguration> resultList = em.createQuery(cq).getResultList();
 
-		return em.createQuery(cq).getSingleResult() > 0;
+		if (resultList.isEmpty()) {
+			return !Arrays.stream(FeatureType.values()).filter(featureType1 -> featureType1.equals(featureType)).findFirst().get().isEnabledDefault();
```

Comment:
this can be replaced with `return featureType. isEnabledDefault()`

Reply:
updated